src/components/common/Tabs/TabContent.tsx

```diff
@@ -6,15 +6,12 @@ type Props = {
   activeTab: number
 }
 
-const TabContent: React.FC<Props> = (props) => {
-  const { tabItems, activeTab } = props
-  return (
-    <div>
-      {tabItems.map((tab) => {
-        return tab.id === activeTab ? tab.content : null
-      })}
-    </div>
-  )
-}
+const TabContent: React.FC<Props> = ({ tabItems, activeTab }) => (
+  <div>
```

Comment:
@W3stside  I see:
- Simple destructuring of props already in the function arguments
- Implicit return value (no need for `return`) 

Thank you.

Reply:
sure. we don't always do it this way and it is actually easier for me to when testing to not use lamda return, but near the end of the pr cycle if you see it doesnt need the curly brace return pattern as it's not calcualting anything in the fn body then it's always nice to clean it up this way. Anyways this part was nothing important more of a reminder that desrtructuring is also common in the fn signature

Reply:
Really appreciate it. Yes the way I understand it, we are defining a component here and we can assume it's implied that this will be the return value when _importing_ it elsewhere.